.graphqlrc.ts

```diff
@@ -27,7 +27,7 @@ module.exports = {
           plugins: [
             'typescript',
             'typescript-operations',
-            'typed-document-node',
+            '@graphql-codegen/typescript-document-nodes',
```

Comment:
I was getting errors when using the linting script. The `TypedDocumentNode` import needs to be a 'type import' instead of a 'normal import' according to the linter. This seems to be caused by the `verbatimModuleSyntax` tsconfig option which was recently added.
 
See https://github.com/voorhoede/head-start/actions/runs/11686385220/job/32542186025 for an example.

This fixes that error.